### ISSUE TITLE
Mass amount of 'undefined' entries shown

### ISSUE THREAD

**Issue by rschiang** [c9s/github-taiwan#1]
I noticed that the gavatars are not showing between #20 - #30 entries, and all user entries over #30 shown as 'undefined'. Is it caused by newly committed script?

Cheers,
RSChiang

**Comment by rschiang** [comment]
![image](https://f.cloud.github.com/assets/1512231/406419/9b923eac-aa8f-11e2-8ca5-9efe11e8a99b.png)

**Comment by c9s** [comment]
yeah , it seems the data is corrupted by GitHub's API rate limitation.

**Comment by rschiang** [comment]
Maybe a community plead to GitHub could bring some difference... or are there workarounds?

**Comment by c9s** [comment]
Currently we just sleep 5 seconds for each requests. but there are only 60 per hour for unauthenticated requests.

so we need to use OAuth API,  for requests using Basic Authentication or OAuth, GitHub limit requests to 5,000 per hour. 

forks are welcome. :)
